orderer/consensus/etcdraft/chain_test.go

```diff
@@ -3073,6 +3073,107 @@ var _ = Describe("Chain", func() {
 			})
 
 			Context("failover", func() {
+				It("should check entries applied to raft cluster", func() {
```

Comment:
I think, it is more appropriate to add this new unit test case under https://github.com/hyperledger/fabric/blob/35e6f5039d0ded7cfbb6e0d6f76125c430d21587/orderer/consensus/etcdraft/chain_test.go#L2978 this context as it is validating the snapshot condition

Reply:
ok.